src/style.scss

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf), url(./fonts/Avenir-Heavy.ttf);
+}
+
+.form {
+  margin: 0;
+  margin-top: 20px;
+}
+
+.cell {
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  transition: all 0.5s;
+  width: 100%;
+
+  &--big {
+    height: 70px;
+    background-image: url(./images/Search.svg);
+    background-repeat: no-repeat;
+    background-position: left 26px top 25px;
+    background-size: 19px;
```

Comment:
Combine with `background` property. Fix all cases

Reply:
all the background properties make like just background?